Simple-Projects/Simple_Projects/Basic/List/main.cpp

```diff
@@ -1,28 +1,31 @@
 #include <iostream>
 #include "List.h"
 
+static void print(const List& list) {
```

Comment:
Even in test code duplication is complication - a simple printing function removed some copy-paste here.